lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -71,8 +72,28 @@ def add_arguments(self, parser):
             default=0,
             type=int,
         )
+        parser.add_argument(
+            '--estimate_first_attempted',
+            help='Use score data to estimate first_attempted timestamp. Values: (true|false|none)',
+            default='none',
+            choices={'true', 'false', 'none', 'True', 'False', 'None'}
+        )
+
+    def _estimate_first_attempted(self, options):
+        """
+        Convert the 'estimate_first_attempted' argument to a boolean or None value.
+        """
+        return {
+            'true': True,
+            'false': False,
+            'none': None,
+        }[options.get('estimate_first_attempted', 'none').lower()]
 
     def handle(self, *args, **options):
+        estimate_first_attempted = self._estimate_first_attempted(options)
+        if isinstance(estimate_first_attempted, bool):
+            waffle().override_for_request(ESTIMATE_FIRST_ATTEMPTED, estimate_first_attempted)
```

Comment:
Fixed.

I updated the task in such a way that we won't run into any backwards compatibility issues, though it occurs to me that we may not have to worry about that in this case.